src/mtype.d

```diff
@@ -3678,17 +3675,17 @@ public:
             case Tcomplex32:
             case Timaginary32:
             case Tfloat32:
-                fvalue = FLT_EPSILON;
+                fvalue = real_t(FLT_EPSILON);
```

Comment:
Is there ever a case where `float_t != float`, same for `double_t`?
I'd agree that using `float[_t].epsilon` would be nicer than keep on using these original C99 macros from `core.stdc.float_`.